### DISCUSSION TITLE
promoter peak

### DISCUSSION THREAD

**Discussion by rojinsafavi** [GreenleafLab/ArchR#631] [Questions / Documentation] [answered]
Hello @jgranja24 @rcorces , 
I was wondering if you could clarify how a peak ( the 500 bp tile) is labeled as a "Promoter" peak?
The reason I am asking this that as far as I know, the promoter is before the starting of the first exon. I looked at a peak that was assigned to the "promoter" type in Genome Browser, and it didn't look like a promoter-type peak to me. I would very much appreciate it 
<img width="1019" alt="Screen Shot 2021-03-18 at 5 13 58 PM" src="https://user-images.githubusercontent.com/19545198/111713760-768f7e80-880d-11eb-8bac-bb6c766566e5.png">

if you could clarify/correct me. thank you!

**Comment by rcorces** [comment, marked as answer]
Full disclosure - there is no perfect way to do this.
From your genome annotations, ArchR knows where the TSS of each gene is located. Of course, some genes may have multiple TSSs but this isnt reflected in TxDb objects and often times a single TSS is reported for each gene symbol.
The end-user controls what is considered the "promoter" region via the `promoterRegion` argument to `addReproduciblePeakSet()`:
https://www.archrproject.com/reference/addReproduciblePeakSet.html

So if you feel like a peak isnt actually a promoter, its probably just because the start site you are looking at is inconsistent with the start site from your `geneAnnotation` object. You can confirm this using the `getGeneAnnotation()` function. In your case, your browser screenshot is looking at GENCODE genes but your geneAnnotation is probably refseq.
